ws-server/src/server.rs

```diff
@@ -26,6 +26,7 @@
 
 use futures_channel::mpsc;
 use futures_util::io::{BufReader, BufWriter};
+use futures_util::stream::StreamExt;
```

Comment:
@niklasad1 I realized that the ws server was using `tokio_stream::StreamExt` and the http server uses `futures_util::stream::StreamExt`. I don't know which is better or if they were using different versions on purpose. This here seems to work though.

Reply:
I think they are different not sure how but but uses the underlying `Stream trait` from futures.

IIRC, it's tokio here because `TcpListenerStream` is used from `tokio_stream` and avoided to another dep on `futures_util`

Reply:
Do you want me to switch back?

Reply:
no, it's fine :)